submissions/chernetskyi8704/Tiny JS World/index.js

```diff
@@ -0,0 +1,199 @@
+class Inhabitant {
+  constructor(Species, Name, Gender, Saying) {
+    this.Species = Species;
+    this.Name = Name;
+    this.Gender = Gender;
+    this.Saying = Saying;
+  }
+
+  addBoldAndRedSemicolon(string) {
+    return string.bold().fontcolor("red");
+  }
+
+  addRedBoldCollon(string) {
+    return string.bold().fontcolor("red");
+  }
+
+  makeStringRedAndBold(string) {
+    return string.bold().fontcolor("Red");
+  }
+
+  makeStringBold(string) {
+    return string.bold();
+  }
+
+  saying() {
+    return ["Saying"].map(
+      (propertyOfTheInhabitant) =>
+        ` ${
+          this.makeStringRedAndBold(propertyOfTheInhabitant) +
+          this.addRedBoldCollon(":")
+        } ${this.makeStringBold(this.Saying)}` +
+        this.addBoldAndRedSemicolon(";")
+    );
+  }
+
+  displayData() {
+    return (
+      ["Species", "Name", "Gender"]
+        .map(
+          (propertyOfTheInhabitant) =>
+            `${
+              this.makeStringRedAndBold(propertyOfTheInhabitant) +
+              this.addRedBoldCollon(":")
+            } ${this[propertyOfTheInhabitant].bold()}`
+        )
+        .join(this.addBoldAndRedSemicolon("; ")) +
+      this.addBoldAndRedSemicolon(";")
+    );
+  }
+}
+class Human extends Inhabitant {
+  constructor(Species, Name, Gender, Saying) {
+    super(Species, Name, Gender, Saying);
+    this.Hands = "2";
+    this.Legs = "2";
+  }
+
+  displayData() {
+    return (
+      `${super.displayData()} ` +
+      ["Hands", "Legs"]
+        .map(
+          (propertyOfTheInhabitant) =>
+            `${
+              this.makeStringRedAndBold(propertyOfTheInhabitant) +
+              this.addRedBoldCollon(":")
+            } ${this[propertyOfTheInhabitant].bold()}`
+        )
+        .join(this.addBoldAndRedSemicolon(";")) +
+      this.addBoldAndRedSemicolon("; ") +
+      this.saying()
+    );
+  }
+}
+
+class Animal extends Inhabitant {}
+
+class Man extends Human {
+  constructor(Name, Gender, Saying) {
+    super("Man🤵", Name, Gender, Saying);
+  }
+}
+
+class Woman extends Human {
+  constructor(Name, Gender, Saying) {
+    super("Woman👩", Name, Gender, Saying);
```

Comment:
Men and women belong to the same species. I just cannot let heresy pass through :)

Reply:
Yeah, that was a stupid one.